modules/Bio/EnsEMBL/Compara/Utils/MasterDatabase.pm

```diff
@@ -811,20 +811,24 @@ sub create_pairwise_wga_mlsss {
 }
 
 sub create_multiple_wga_mlsss {
-    my ($compara_dba, $method, $species_set, $with_gerp, $source, $url) = @_;
+    my ($compara_dba, $method, $species_set, $with_gerp, $no_release, $source, $url) = @_;
 
     my @mlsss;
     push @mlsss, create_mlss($method, $species_set, $source, $url);
+    $mlsss[-1]->{_no_release} = ($no_release || 0);
     if ($with_gerp) {
         my $ce_method = $compara_dba->get_MethodAdaptor->fetch_by_type('GERP_CONSTRAINED_ELEMENT');
         push @mlsss, create_mlss($ce_method, $species_set, $source, $url);
+        $mlsss[-1]->{_no_release} = ($no_release || 0);
         my $cs_method = $compara_dba->get_MethodAdaptor->fetch_by_type('GERP_CONSERVATION_SCORE');
         push @mlsss, create_mlss($cs_method, $species_set, $source, $url);
+        $mlsss[-1]->{_no_release} = ($no_release || 0);
     }
     if ($method->type eq 'CACTUS_HAL') {
         my $pw_method = $compara_dba->get_MethodAdaptor->fetch_by_type('CACTUS_HAL_PW');
         push @mlsss, @{ create_mlsss_on_pairs($pw_method, $species_set->genome_dbs, $source, $url) };
-    }
+        $mlsss[-1]->{_no_release} = ($no_release || 0);
+    }    
```

Comment:
Yes, I have refactored it so that it won't miss mlsses. 

I don't see a reason to store a tag, but equally I don't see a reason not to, so I'll leave the final say to @muffato ..

Reply:
I like the idea of storing a `no_release` tag in the database. We can know which mlss are not released because of this XML flag. Could be helpful to track down bugs in `make_object_current` and `retire_object`

Reply:
Ah, wait, I see that create_all_mlss.pl already understands the hash field `$mlss->{_no_release}` (was meant to be used for the polyploid genomes). Here you're extending the _no_release scheme to some EPO alignments. In this case, to simplify the PR, I would say there is no compelling reason to move over to tags. Sorry !